lib/ansible/plugins/terminal/ce.py

```diff
@@ -31,7 +31,13 @@ class TerminalModule(TerminalBase):
         re.compile(br'[\r\n]?<.+>(?:\s*)$'),
         re.compile(br'[\r\n]?\[.+\](?:\s*)$'),
     ]
+    #: terminal initial prompt
+    #: The password needs to be changed. Change now? [Y/N]:
+    terminal_initial_prompt = br'Change\s*now\s*\?\s*\[Y\/N\]\s*:'
```

Comment:
In the initial connection, there may be prompts to change the password.
But password modification is not allowed.and password modification requires interaction.
So do not!